getting_started/build-guide/build-from-scratch.rst

```diff
@@ -59,7 +61,6 @@ Replace the default Ubuntu alsa-lib with the one we just built.
 .. code-block:: bash
 
    $ sudo cp /usr/lib/libasound.*    /usr/lib/x86_64-linux-gnu/
-   $ sudo cp /usr/lib/alsa_lib/*    /usr/lib/x86_64-linux-gnu/alsa-lib
```

Comment:
this will essentially prevent pulseaudio from working on the host machine. It's better to use LD_LIBRARY_PATH=<path-to-alsa-lib>/src/.libs:$LD_LIBRARY_PATH.

Reply:
HI Pierre,

Changing the line is not big deal. Trying to clarify your comments. You meant removing both lines and adding exporting LD_LIBRARY is better way even considering PulseAudio? Bear with me, I have little idea of PulaseAudio. I appreciate your comments but not sure why this will break PulseAudio. Could you explain a bit more?

Reply:
The point was to avoid breaking the host setup with a wild install of alsa-lib, so yes the two lines should be removed.
There are two ways to make sure the latest alsa-lib is used:
1) don't install but only use an environment variable to refer to the latest alsa-lib
2) build a debian/rpm package and install it cleanly.

Reply:
Now 100% clear to me. new commit is made.

https://github.com/thesofproject/sof-docs/pull/74/commits/93948b8af7c5c13c79351013a87c89a892eed4ba